requirements.txt

```diff
@@ -6,7 +6,8 @@ pyparsing<3.0.0
 zipp<2.0.0
 pywinrm==0.2.2
 requests==2.24.0
-paramiko~=2.10.0
+# TODO: vagrant with paramiko 2.10
+paramiko~=2.7.2
```

Comment:
is this roleback on purpose?

Reply:
nope, this is an artifact of paramiko 2.10 not working with ducktape's vagrant setup for some reason. I'll revert this change.

Reply:
done